include/git2cpp/blame.h

```diff
@@ -1,23 +1,25 @@
-﻿#pragma once
+#pragma once
 
-#include <memory>
+#include <git2/blame.h>
 
-struct git_blame;
-struct git_blame_hunk;
+#include <memory>
 
 namespace git
 {
     struct Blame
     {
+        uint32_t hunk_count() const;
+
+        const git_blame_hunk * hunk_byindex(uint32_t index) const;
+
+        const git_blame_hunk * hunk_byline(size_t lineno) const;
```

Comment:
I'm OK without the prefix `get_`, but examples/blame.cpp should be fixed in this case.

Reply:
I'll fix the example.